gulpfile.js

```diff
@@ -31,24 +31,30 @@ gulp.task('clean', function () {
 /**
  * Construction du dossier de build
  */
-gulp.task('build', ['copybase', 'copyreveal'], function () {
+gulp.task('build', ['copyAppYaml'], function () {
 
   // Remplacement du placeholder par le nom de la formation
   return gulp.src([path.join(fwkConfig.frameworkPath, 'index.html')])
     .pipe(g.replace(/(FORMATION_NAME)/g, fwkConfig.slidesPdfName))
     .pipe(gulp.dest(fwkConfig.outputPath));
 });
 
+gulp.task('copyAppYaml', ['copybase', 'copyreveal', 'copyslides'], function() {
```

Comment:
This task is named `copyAppYaml` but it actually copies everything. Maybe this task should depend on nothing and `build` should depend on `['copyAppYaml', 'copybase', 'copyreveal', 'copyslides']`.